File: libs/LSPs/projects/python_project/src/utils/string_utils.py
def reverse_string(text: str) -> str:
    """
    Reverse the given string.
    
    Args:
        text: The string to be reversed.
    
    Returns:
        The reversed string.
    """
    if not isinstance(text, str):
        raise TypeError("Input must be a string")
    return text[::-1]

undefined_var = some_var 

File: libs/LSPs/projects/ts_project/src/utils/stringUtils.ts
/**
 * Reverses the characters in a string.
 *
 * @param text - The string to reverse
 * @returns The reversed string
 * @throws {Error} If the input is not a string
 *
 * @example
 * ```ts
 * reverseString('hello') // returns 'olleh'
 * reverseString('TypeScript') // returns 'tpircSepyT'
 * ```
 */
export function reverseString(text: string): string {
    if (typeof text !== 'string') {
        throw new Error('Input must be a string');
    }
    return text.split('').reverse().join('');
}